src/modules/auth/login.controller.ts

```diff
@@ -132,4 +132,31 @@ export class LoginController {
 
     return res.send({ token, refreshToken });
   }
+
+  @Get('auth/status')
+  async status(@Req() req: Request) {
+    const accessTokenString =
+      req.headers['authorization']?.replace('Bearer ', '') ||
+      req.cookies[this.configService.get<string>('JWT_ACCESS_TOKEN_NAME')];
+
+    if (!accessTokenString) {
+      return {
+        user: null,
+      };
+    }
+
+    try {
+      const tokenData = await this.tokenService.verifyAccessToken(
+        accessTokenString,
+      );
+
+      return {
+        user: tokenData?.did || null,
+      };
+    } catch {
+      return {
+        user: null,
+      };
+    }
```

Comment:
Why the `try/catch` here? Are we expecting errors from `verifyAccessToken` in some cases?
```suggestion
      const tokenData = await this.tokenService.verifyAccessToken(
        accessTokenString,
      );

      return {
        user: tokenData?.did || null,
      };
      return {
        user: null,
      };
```

Reply:
`verifyAccessToken` will throw an error when token is invalid, expired etc.

Reply:
Ok, thanks for the explanation. IMO, could be good to have a comment to explain that but it's fine for now 😄